src/main/java/org/sasanlabs/service/vulnerability/sampleVulnerability/SampleVulnerability.java

```diff
@@ -0,0 +1,109 @@
+package org.sasanlabs.service.vulnerability.sampleVulnerability;
+
+import org.sasanlabs.internal.utility.LevelConstants;
+import org.sasanlabs.internal.utility.Variant;
+import org.sasanlabs.internal.utility.annotations.AttackVector;
+import org.sasanlabs.internal.utility.annotations.VulnerableAppRequestMapping;
+import org.sasanlabs.internal.utility.annotations.VulnerableAppRestController;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.sasanlabs.vulnerability.types.VulnerabilityType;
+import org.springframework.web.bind.annotation.RequestParam;
+
+/**
```

Comment:
please remove samplevulnerability related code.

Reply:
Thanks, I have already removed it